plugins/openstack/keystone/check_keystone-api.sh

```diff
@@ -38,9 +38,10 @@ usage ()
     echo " -H <Auth URL>    URL for obtaining an auth token"
     echo " -U <username>    Username to use to get an auth token"
     echo " -P <password>    Password to use ro get an auth token"
+    echo " -T <seconds>     Timeout after integer <seconds>"
```

Comment:
Hrm. Okay. How about $WAIT_FOR_REPLY? The "-T" could be changed to "-W" and usage "Time, in seconds, to wait for reply (integer value only)"?